### DISCUSSION TITLE
[EMSUSD-1030] Create USD Parent Scope setting doesn't propagate from Options to File dialog

### DISCUSSION THREAD

**Discussion by Ark-Dem** [Autodesk/maya-usd#3617] [General] [answered]
(Maya 2024.2, Win10, mayaUSD 0.27.0)

When I press File>Export All/Selected (Options) options menu pops up. I make my settings and press "Export" button - file dialog pops up with the same options menu embedded to the right... but specifically "Create USD Parent Scope" setting gets reverted to default (empty text field). Other settings seem to propagate correctly from (Options) dialog.

Not a big deal - I have to either set it twice or avoid using (Options) dialog at all and go straight into file dialog instead... But it would be nice to fix this to avoid confusions.

**Comment by wallworm** [comment]
Thanks for reporting.

**Comment by womanyoyoyo-adsk** [comment, marked as answer]
Hi @Ark-Dem, thanks for bringing this up. This is important because the user exports and doesn't get the default prim he first inputted. I've logged this internally.
